cookbooks/aws-parallelcluster-platform/resources/dcv/partial/_ubuntu_common.rb

```diff
@@ -82,6 +82,62 @@ def optionally_disable_rnd
     end
   end
 
+  # Disable Wayland in GDM to ensure Xorg is used
+  # This is required for Ubuntu 22.04+ where Wayland is the default
+  # Without this, GDM won't start Xorg on headless GPU instances
+  def disable_wayland
+    bash 'Disable Wayland in GDM' do
+      user 'root'
+      code <<-DISABLEWAYLAND
+        set -e
+        if [ -f /etc/gdm3/custom.conf ]; then
```

Comment:
Does this file exist if GDm3 is being used?

Reply:
Also can we use a template instead of using Bash?

Reply:
As discussed on call, we will not move the changes to disable Wayland for all OS and will keep it just for Ubuntu since the issue has happened only with the DLAMI's that we are testing.